src/main/resources/assets/reignofnether/lang/en_us.json

```diff
@@ -1,42 +1,677 @@
+    "server.reignofnether.match_reset": "Match has been reset!",
+    "server.reignofnether.match_locked": "The RTS match has been locked. No new players may join.",
+    "server.reignofnether.match_unlocked": "The RTS match has been unlocked. New players may now join.",
+    "server.reignofnether.sync_enabled": "RTS login-syncing has been enabled.",
+    "server.reignofnether.sync_disabled": "RTS login-syncing has been disabled.",
+    "server.reignofnether.enabled_fog_of_war": "enabled fog of war for all players",
+    "server.reignofnether.disabled_fog_of_war": "disabled fog of war for all players",
+    "server.reignofnether.locked": "This match has been locked from new RTS players joining.",
+    "server.reignofnether.already_started": "You already started your RTS match!",
+    "server.reignofnether.outside_border": "Cannot start outside map border",
+    "server.reignofnether.started": "%s has started their game!",
+    "server.reignofnether.total_players": "There are now %d total RTS player(s)",
+    "server.reignofnether.bot_added": "%s (bot) has been added to the game!",
+    "server.reignofnether.used_cheat": "%s used cheat: %s %s",
+    "server.reignofnether.enabled_cheat": "%s enabled cheat %s",
+    "server.reignofnether.disabled_cheat": "%s disabled cheat %s",
+    "server.reignofnether.all_cheats": "%s enabled all cheats",
+    "server.reignofnether.revealed": "%s has not rebuilt their capitol and is being revealed!",
+
+    "server.resources.reignofnether.not_enough_food": "You don't have enough food",
+    "server.resources.reignofnether.not_enough_wood": "You don't have enough wood",
+    "server.resources.reignofnether.not_enough_ore": "You don't have enough ore",
+    "server.resources.reignofnether.not_enough_pop": "You don't have enough population supply",
+    "server.resources.reignofnether.max_pop": "You have reached the maximum population",
+
+    "research.reignofnether.upgrade_completed": "Upgrade completed: %s",
+    "research.reignofnether.advanced_portals": "Advanced Portals",
+    "research.reignofnether.advanced_portals.tooltip1": "Allows portals to be constructed off of nether terrain",
+    "research.reignofnether.advanced_portals.tooltip2": "and enables them to be upgraded into transport portals,",
+    "research.reignofnether.advanced_portals.tooltip3": "which can instantly teleport units between them.",
+    "research.reignofnether.blaze_firewall": "Blaze Firewall",
+    "research.reignofnether.blaze_firewall.tooltip1": "Allows Blazes to fire a single shot that ignites all",
+    "research.reignofnether.blaze_firewall.tooltip2": "blocks in a line, but disables their attacks temporarily.",
+    "research.reignofnether.brute_shields": "Brute Shields",
+    "research.reignofnether.brute_shields.tooltip1": "Allows Brutes to raise a shield to reduce projectile ",
+    "research.reignofnether.brute_shields.tooltip2": "damage taken by 67%% and movement speed by 50%%.",
```

Comment:
`"research.reignofnether.brute_shields.tooltip2": "damage taken by 67% and movement speed by 50%.",`

Reply:
This one's intentional, localizations are formatted and % must be escaped